### PR TITLE
Improve DX: added yaml service definition

### PR DESCRIPTION
**Pull request by borNfreee**
Updated cookbook with included service definition for provided listener class

### CODE REVIEW

File: Resources/doc/cookbook/custom-properties.md [hunk 1]
```diff
@@ -31,3 +31,11 @@ class CustomPropertyListener implements EventSubscriberInterface
     }
 }
 ```
+
+Service definition:
+```yml
+acme.listener.custom_property:
+    class: AcmeBundle\EventListener\CustomPropertyListener
```
Comment (davidfuhr) [inline Resources/doc/cookbook/custom-properties.md]: You should also add a namespace definition to the php code above to make it consistent.

Comment (borNfreee) [inline Resources/doc/cookbook/custom-properties.md]: fixed

**Comment by borNfreee** [comment]
ping @merk

**Comment by merk** [comment]
Thanks

**Pull request merged**